tests/test_lora.py

```diff
@@ -232,9 +242,9 @@ def test_lora_linear_utilization(apply_to, target_layer_names, mlp_class_name):
             assert layer_name + lora_sublayer in state_dict
 
     # check that only target layers have LoRA weights
-    for key in state_dict:
-        if key.endswith(lora_sublayers):
-            assert key.startswith(target_layer_names)
+    lora_params = [k for k in state_dict if k.endswith(lora_sublayers)]
+    lora_params = {k[:-7] for k in lora_params}
+    assert lora_params == target_layer_names
```

Comment:
Just a question: you decided to stick with this approach that utilizes a magic number so you can see a better output in case if assertion fails?

Reply:
That's right. It's equivalent to what was before. The magic number is `len(".lora_A")`